src/flexalloc_xnvme_env.c

```diff
@@ -192,61 +192,60 @@ fla_xne_sync_seq_w_naddrs(struct xnvme_dev * dev, const uint64_t slba, const uin
 // Read or write out a larger chunk striped across multiple objects
 int
 fla_xne_sync_strp_seq_x(struct xnvme_dev *dev, const uint64_t offset, uint64_t nbytes,
-                        void const *buf, struct fla_sync_strp_params *sp, bool write)
+                        void const *buf, struct fla_sync_strp_params const * const sp, bool write)
 {
+  int err;
   uint32_t nsid = xnvme_dev_get_nsid(dev);
   struct xnvme_queue *queue = NULL;
-  int err;
   struct fla_async_cb_args cb_args = { 0 };
   const struct xnvme_geo *geo = xnvme_dev_get_geo(dev);
-  uint32_t strp_nobjs = sp->strp_nobjs;
-  uint32_t strp_nbytes = sp->strp_nbytes;
-  uint64_t obj_len = sp->obj_len;
-  uint32_t strp_blks = (strp_nbytes / geo->lba_nbytes) - 1;
-  uint64_t bytes_xfer = 0, strp_offset = offset;
+  uint32_t strp_blks = (sp->strp_nbytes / geo->lba_nbytes) - 1;
+  uint64_t bytes_to_xfer = nbytes, strp_offset = offset, strp_to_xfer;
   char *strp_buf = (char *)buf;
 
-  // Nbytes must be a multiple of strp_nobjs * strp_nbytes
-  if (nbytes % (strp_nobjs * strp_nbytes))
+  // Nbytes must be a multiple of sp->strp_nobjs * strp_nbytes
+  /*if (nbytes % (sp->strp_nobjs * sp->strp_nbytes))
```

Comment:
I cannot find these comments. maybe its a previous version of the patch?

Reply:
Found in second commit, when reviewing commits in order. I'm getting used to the GUI, I'm afraid it might attach comments to the whole series. Ideally I could make comments on each individual patch.

Reply:
Do them as you prefer. It should be that a comment on a commit is also valid for the whole series. If it is not then it is probably a mistake on my part and it is something that should have not gone into the subcommit in the first place.